packages/app/src/vis-packs/core/compound/CompoundMatrixVisContainer.tsx

```diff
@@ -17,19 +20,32 @@ function CompoundMatrixVisContainer(props: VisContainerProps) {
   assertCompoundType(entity);
   assertPrintableCompoundType(entity);
 
+  const { shape: dims } = entity;
+  const axesCount = Math.min(dims.length, 1);
```

Comment:
Don't think you need `Math.min`, since `assertArrayShape` guarantees a shape great than 0: `dataset.shape.length > 0` (and also the call to `hasMinDims` when checking for supported visualisations).

Reply:
:fishing_pole_and_fish: